src/network_analysis.jl

```diff
@@ -721,7 +721,8 @@ end
     iscomplexbalanced(rs::ReactionSystem, parametermap)
 
 Constructively compute whether a network will have complex-balanced equilibrium
-solutions, following the method in van der Schaft et al., [2015](https://link.springer.com/article/10.1007/s10910-015-0498-2#Sec3). Accepts a dictionary, vector, or tuple of variable-to-value mappings, e.g. [k1 => 1.0, k2 => 2.0,...]. 
+solutions, following the method in van der Schaft et al., [2015](https://link.springer.com/article/10.1007/s10910-015-0498-2#Sec3). 
+Accepts a dictionary, vector, or tuple of variable-to-value mappings, e.g. [k1 => 1.0, k2 => 2.0,...]. 
```

Comment:
@vyudu This and the next doc string were very long and a single line. I broke them up so that each line is no more than 100 characters (what I try to limit myself to). This should make them more readable when browsing the code. Is this OK with you?

Reply:
Yes! Will keep this in mind.